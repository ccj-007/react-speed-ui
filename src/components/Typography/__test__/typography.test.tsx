import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Typography } from '../../index';

describe('test Typography component', () => {
  it('should render', () => {
    const { asFragment } = render(
      <>
        <Typography.Text />
        <Typography.Title />
      </>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
