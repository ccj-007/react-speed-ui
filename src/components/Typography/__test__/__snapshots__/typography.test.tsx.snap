// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Typography component should render 1`] = `
<DocumentFragment>
  <span
    class="text text-default"
  />
  <h6 />
</DocumentFragment>
`;
